app/src/lib/stats/stats-database.ts

```diff
@@ -227,6 +253,11 @@ export interface IDailyMeasures {
    * a keyboard shortcut only `createPullRequestCount` will increment.
    */
   readonly suggestedStepCreatePullRequest: number
+
+  /** The number of times the user has used the view stash action from
```

Comment:
:art: The first line of a multi-line JSDoc comment should be empty - nudge this down to it's own line